jinja2_standalone_compiler/__init__.py

```diff
@@ -4,14 +4,23 @@
 import os
 import re
 import sys
-
 import click
+from colorama import init, Fore, Style
```

Comment:
Does it really matter when receiving the lib trought pip? It should install colorama, no? How can make an optional requirement? I think I need colorama in requiremets.txt to build the egg, right?